lib/logflare/source/bigquery/buffer_counter.ex

```diff
@@ -177,24 +177,26 @@ defmodule Logflare.Source.BigQuery.BufferCounter do
   end
 
   defp broadcast_buffer(state) do
-    local_buffer = %{Node.self() => %{len: state.len}}
+    Task.start(fn ->
```

Comment:
Can use the Logflare Task Supervisor instead of the generic Task, so that it is supervised

Also, we can do async_nolink so that caller is not affected by task fails.